client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigatedPersonInfo/InvestigatedPersonInfo.tsx

```diff
@@ -174,7 +174,7 @@ const InvestigatedPersonInfo = (props: Props) => {
                                             test-id='currentSubStatus'
                                             options={subStatuses}
                                             getOptionLabel={(option) => option}
-                                            inputValue={investigationStatus.subStatus as string | undefined}
+                                            inputValue={investigationStatus.subStatus || '' as string | undefined}
```

Comment:
yes because it could be null and inputValue doesnt except it
![image](https://user-images.githubusercontent.com/42100027/98834974-84d92280-2448-11eb-854f-10df010c5441.png)

Reply:
But u make it a string anyway

Reply:
Cause it can be a string... 
The value can be null but the autocomplete field doesn't except null, so if it's null convert it to an empty string